### PR TITLE
github.com/Free-TV/IPTV and github.com/iptv-org/iptv providers

### PR DESCRIPTION
**Pull request by antonydp**
https://github.com/Free-TV/IPTV and https://github.com/iptv-org/iptv providers

### CODE REVIEW

File: IptvorgProvider/src/main/kotlin/com/lagradost/IptvorgProvider.kt [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+import com.lagradost.cloudstream3.*
+import com.lagradost.cloudstream3.utils.AppUtils.parseJson
+import com.lagradost.cloudstream3.utils.AppUtils.toJson
+import com.lagradost.cloudstream3.utils.ExtractorLink
+import com.lagradost.cloudstream3.utils.Qualities
+import java.io.InputStream
+
+class IptvorgProvider : MainAPI() {
+    override var lang = "en"
+    override var mainUrl = "https://github.com/iptv-org/iptv"
```
Comment (C10udburst) [inline IptvorgProvider/src/main/kotlin/com/lagradost/IptvorgProvider.kt:12]: maybe use `https://raw.githubusercontent.com/iptv-org/iptv/master/README.md` instead for the main url?

File: FreeTVProvider/src/main/kotlin/com/lagradost/FreeTVProviderPlugin.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+package com.lagradost
+
+import com.lagradost.cloudstream3.plugins.CloudstreamPlugin
+import com.lagradost.cloudstream3.plugins.Plugin
+import android.content.Context
+
+@CloudstreamPlugin
+class TvItalianaProviderPlugin: Plugin() {
```
Comment (C10udburst) [inline FreeTVProvider/src/main/kotlin/com/lagradost/FreeTVProviderPlugin.kt:9]: very minor nitpick, but this should probably be called `FreeTVProviderPlugin`

**Review by C10udburst — CHANGES_REQUESTED** [review]

File: FreeTVProvider/src/main/kotlin/com/lagradost/FreeTVProviderPlugin.kt (hunk 2 above)
Comment (antonydp) [reply FreeTVProvider/src/main/kotlin/com/lagradost/FreeTVProviderPlugin.kt:9]: yes it was a typo, thanks

File: FreeTVProvider/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+     * 1: Ok
+     * 2: Slow
+     * 3: Beta only
+     * */
+    status = 1 // will be 3 if unspecified
+    tvTypes = listOf(
+        "Live",
+    )
+
+    iconUrl = "https://www.google.com/s2/favicons?domain=github.com&sz=24"
```
Comment (Blatzar) [inline FreeTVProvider/build.gradle.kts:23]: We updated the support for size, try something like
iconUrl = "https://www.google.com/s2/favicons?domain=github.com&sz=%size%"

File: FreeTVProvider/build.gradle.kts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+// use an integer for version numbers
+version = 1
+
+
+cloudstream {
+    // All of these properties are optional, you can safely remove them
+
+    // description = "Lorem Ipsum"
+    // authors = listOf("Cloudburst")
```
Comment (Blatzar) [inline FreeTVProvider/build.gradle.kts:9]: feel free to leave your name here :+1: 
While not visible in the app right now it probably will be in the future

File: FreeTVProvider/build.gradle.kts (hunk 3 above)
Comment (antonydp) [reply FreeTVProvider/build.gradle.kts]: done, thanks!

**Review by C10udburst — APPROVED** [review]

**Pull request merged**